lib/react-dom.js

```diff
@@ -215,17 +213,25 @@ declare class SyntheticMouseEvent<
   screenX: number;
   screenY: number;
   shiftKey: boolean;
+|}
+
+declare class SyntheticMouseEvent<
+  +T: EventTarget = EventTarget,
+> extends SyntheticUIEvent<T, MouseEvent> {
+  ...$SyntheticMouseEvent,
```

Comment:
Spread in classes?

Reply:
🤦‍♀️ good point. Alternatively, I was thinking about introducing second param to `SyntheticMouseEvent` that would accept `Event`, and will default to `MouseEvent`

Reply:
maybe `mixins`?
```
declare class A {
  a: string
}
declare class C {
  b: string
}
declare class B extends C mixins A {}

const b = new B()

b.a;
b.b;
```